### DISCUSSION TITLE
I have a question about the components of the wizard.

### DISCUSSION THREAD

**Discussion by tomokusaba** [microsoft/fluentui-blazor#1284] [General]
Why is the primary button placed on the right side?
Normally, FluentUI says that buttons with accent colors are placed on top or left.
However, in the wizard component, the Next button is placed on the right side and is an accent color.
Why is this so designed?

**Comment by dvoituron** [comment]
The buttons for moving from one step to the next are usually placed at the bottom.
Here are a few examples from other libraries (not all FluentUI, of course).
This is also the case with the Wizards used by Azure.

- Telerik: https://demos.telerik.com/blazor-ui/wizard/overview
- MudBlazor: https://codebeam-mudextensions.pages.dev/mudstepper

On the other hand, buttons are sometimes placed on the left or right. We've decided to place them on the right by default, 
but all this is fully configurable, as shown in the documentation: https://www.fluentui-blazor.net/Wizard#customized

We already presented the design a few weeks ago in the issue #1117
